src/content/en/tools/chrome-devtools/webauthn/index.md

```diff
@@ -0,0 +1,92 @@
+project_path: /web/tools/chrome-devtools/_project.yaml
+book_path: /web/tools/chrome-devtools/_book.yaml
+description: Emulate Authenticators and Debug WebAuthn in Chrome DevTools.
+
+{# wf_updated_on: 2020-09-27 #}
+{# wf_published_on: 2020-09-28 #}
+{# wf_blink_components: Security, Platform>DevTools #}
+
+# Emulate Authenticators and Debug WebAuthn in Chrome DevTools {: .page-title }
+
+{% include "web/_shared/contributors/fawazm.html" %}
+{% include "web/_shared/contributors/jecelynyeen.html" %}
+
+Use the **WebAuthn** tab in Chrome DevTools to create and interact with software-based virtual authenticators. 
+
+
+## Open the WebAuthn tab  {: .open }
+
+1. Visit a page that uses WebAuthn, such as our demo page here - [webauthndemo.appspot.com/](https://webauthndemo.appspot.com/) (please login to access the page).
+2. [Open DevTools](/chrome-devtools/open).
+3. Click the **More Options** 
+![More](/web/tools/chrome-devtools/images/shared/more.png){: .inline-icon } > **More tools** > **WebAuthn** to open the WebAuthn tab.
+
+![WebAuthn tab](/web/tools/chrome-devtools/webauthn/images/01-webauthn-tab.png)
+
+
+## Enable the virtual authenticator environment {: .enable }
+
+1. On the WebAuthn tab, click to enable the checkbox **Enable virtual authenticator environment**.
+2. Once enabled, you will see a new section **New authenticator**.
+
+![Enable virtual authenticator environment](/web/tools/chrome-devtools/webauthn/images/02-enable-virtual-auth.png)
+
+
+## Add a virtual authenticator {: .add }
+
+1. On the **New authenticator** section, configure the options.
+2. Click on the **Add** button. 
+3. You can now see a new section of your newly-created authenticator.
+![Authenticator](/web/tools/chrome-devtools/webauthn/images/03-authenticor.png)
+
+The **Authenticator** section includes a **Credentials** table. The table will empty until a credential is registered to the authenticator.
```

Comment:
```suggestion
The **Authenticator** section includes a **Credentials** table. The table is empty until a credential is registered to the authenticator.
```

(or "will be", but in general it's nice to [avoid “will”](https://twitter.com/mathias/status/839160485324947456))